testing/scripts/seldon_e2e_utils.py

```diff
@@ -418,3 +436,49 @@ def clean_string(string):
     string = re.sub(r"\]$", "", string)
     string = re.sub(r"[_\[\./]", "-", string)
     return string
+
+
+def assert_model_during_op(op, *assert_args, **assert_kwargs):
+    with ThreadPoolExecutor(max_workers=1) as executor:
+        future = executor.submit(op)
+
+        try:
+            while future.running():
+                assert_model(*assert_args, **assert_kwargs)
```

Comment:
Hmm.. this is a full speed loop - it could crash just from lack of resources.
Maybe add some short sleep after each assertion? Like 100 or 50 ms?

Reply:
I thought about it. The thing is that we do want to continuously keep making requests to the model. The code inside `assert_model()` does take a bit of time so we should be covered by that.

Reply:
Cool, trust you on this one!